script.js

```diff
@@ -1 +1,125 @@
-//CEOS 19기 프론트엔드 파이팅🔥 ദ്ദി˶ˊᵕˋ˵)
+const inputField = document.querySelector('.todo-input'); //todo 입력창
+const TodoButton = document.querySelector('.todo-input-button'); //+추가 버튼
+const todoLists = document.querySelector('.todo-lists'); //todo list목록
+const doneLists = document.querySelector('.done-lists'); //done list목록
+const date = document.querySelector('h1#date'); //날짜 표기
+const week = document.querySelector('h3#week'); //요일 표기
+const showTodoNumber = document.querySelector('.show-todo-number'); //todo list div 개수
+const showDoneNumber = document.querySelector('.show-done-number'); //done list div 개수
+
+
+//lists 개수 세기
+function todoListsNumber(){
+    const todoNumber = todoLists.getElementsByTagName("div");
+    const doneNumber = doneLists.getElementsByTagName("div");
+    
+    showTodoNumber.innerText = `${todoNumber.length} lists to do`;
+    showDoneNumber.innerText = `${doneNumber.length} lists are done! Way to go : )`
+}
+todoListsNumber();
+
+
+//+ 버튼 클릭시 todo에 list를 추가
+TodoButton.addEventListener('click', addTodo); 
+function addTodo(event){
+    event.preventDefault(); //event에 대한 기본 동작 실행 방지
+    
+    //todo list div 생성
+    const todoDiv = document.createElement("div");
+    todoDiv.classList.add("todo-div");
+
+    //todo list li 생성
+    const todoLi = document.createElement("li");
+    todoLi.classList.add("todo-li");
+   
+   
+    //todo list 삭제 button 생성
+    const deleteButton = document.createElement('button');
+    deleteButton.innerHTML = '<i class="fa-solid fa-circle-minus" style="color: #ff5f58;"></i>'
+    deleteButton.classList.add('delete-button');
+   
+     //todo list done button 생성
+     const doneButton = document.createElement('button');
+     doneButton.innerHTML = '<i class="fa-solid fa-circle-check" style="color: #28c840;"></i>';
+     doneButton.classList.add("done-button");
+
+
+    //공백 여부에 따른 처리
+    if(!inputField.value.trim())
+            alert("할일을 추가해 보세요!");
+    else{
+        todoLi.innerText = inputField.value;
+        todoDiv.appendChild(todoLi);
+        todoDiv.appendChild(deleteButton);
+        todoDiv.appendChild(doneButton);
+        todoLists.appendChild(todoDiv);
+        todoListsNumber();
+    }
+
+    //input field 초기화
+    inputField.value = '';
+}  
+
+//enter키 입력시 + 버튼 누른 효과
+inputField.addEventListener("keyup", function (event) {
+    if (event.keyCode === 13) {
```

Comment:
`keyCode`는 [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) 되어서 `key` 속성을 사용하는 건 어떨까요 ??
```suggestion
    if (event.key === "Enter") {
```